src/zenml/orchestrators/base_orchestrator.py

```diff
@@ -110,6 +111,23 @@ def _patched_remove_stateful_working_dir(stateful_working_dir: str) -> None:
             stateful_working_dir,
         )
 
+def _patched_remove_output_dirs(output_dict: Dict[str, List[types.Artifact]]) -> None:
+  """Remove dirs of output artifacts' URI.
+  
+  Args:
+    output_dict: Dictionary of strings to output artifacts.
+  """
+  # The original implementation was doing a fileio.rmtree 
+  # without checking for the existence of the object. This 
+  # caused a cascading failure, and users of ZenML then see 
+  # an internal TFX error in some cases.
+  for _, artifact_list in output_dict.items():
+    for artifact in artifact_list:
+        if fileio.isdir(artifact.uri):
+            fileio.rmtree(artifact.uri)
+        elif fileio.exists(artifact.uri):
+            fileio.remove(artifact.uri)
```

Comment:
```suggestion
  for artifact in output_dict.values():
      if fileio.isdir(artifact.uri):
          fileio.rmtree(artifact.uri)
      elif fileio.exists(artifact.uri):
          fileio.remove(artifact.uri)
```

Perhaps?

Reply:
I think you missed a for loop

Reply:
Oh, yes you're perfectly right. It's probably clearer to do it the way you originally wrote it vs switching out `items` for `values` on line 124.

Reply:
Could you however do me a favor and format the whole thing with our script? :-) I dont have it set up here. Feel free to push